test/e2e/config/ibmcloud-e2e-powervs.yaml

```diff
@@ -40,9 +40,11 @@ providers:
         targetName: "metadata.yaml"
       - sourcePath: "${PWD}/templates/cluster-template-powervs.yaml"
         targetName: "cluster-template-powervs.yaml"
+      - sourcePath: "${PWD}/test/e2e/data/templates/cluster-template-md-remediation.yaml"
+        targetName: "cluster-template-md-remediation.yaml"
```

Comment:
instead of adding an entire template, is there a way to just mention MHC only so that we don't repeat the above mentioned template from `${PWD}/templates/cluster-template-powervs.yaml`

Reply:
Yes that can be done via `Kustomize`, need to reorganise few things in e2e structure. We can merge this PR for now, have discussed with @Amulyam24 she will create a issue and work on reorganising it post merge.

Reply:
> Yes that can be done via `Kustomize`, need to reorganise few things in e2e structure. We can merge this PR for now, have discussed with @Amulyam24 she will create a issue and work on reorganising it post merge.

Lets target this by end of this release.